tests/row_association/test_row_association.py

```diff
@@ -77,3 +110,16 @@ def test_empty_line_items_init(three_row_invoice_preds, three_row_invoice_tokens
     litems.assign_row_number()
     grouped_rows = litems.get_line_items_in_groups()
     assert len(grouped_rows) == 3
+
+
+def test_assign_row_number_order(
+    three_row_bank_statement_preds, three_row_bank_statement_tokens
```

Comment:
confirming that without the change you made, this would have only been 1 row or something else? do we need to be checking what is contained within these two rows?

Reply:
I would change the name of this test to state the problem this is testing for, i.e. sort_bbtop

Reply:
if there should only be 'two_rows' jsons, shouldn't be called 'three_row....'